CityOfDetroit/bloom
# Feature/phone number account creation

## Issue

- Addresses #701

## Description

Add a phone number field to the account creation page, and save it to the database.

Desktop:
![localhost_3000_create-account](https://user-images.githubusercontent.com/66751489/140830480-cf3f432e-e7c8-4001-abb3-48c3b811a871.png)

Mobile:
![localhost_3000_create-account (3)](https://user-images.githubusercontent.com/66751489/140830520-a81f9e1b-087e-4307-854b-a015b4bd42ef.png)

## Type of change


## How Can This Be Tested/Reviewed?

Create a new account with a phone number set, and check it's set in the database via `select phone_number from user_accounts where email='your@email';`

## Checklist:
